src/common/syncjournaldb.cpp

```diff
@@ -1494,6 +1506,7 @@ void SyncJournalDb::setUploadInfo(const QString &file, const SyncJournalDb::Uplo
         _setUploadInfoQuery->bindValue(4, i._errorCount);
         _setUploadInfoQuery->bindValue(5, i._size);
         _setUploadInfoQuery->bindValue(6, i._modtime);
+        _setUploadInfoQuery->bindValue(7, i._contentChecksum);
```

Comment:
There is a `operator==` for UploadInfo that you probably want to adjust too (or remove entirely!)